tests/unit/test_connection_overrides.py

```diff
@@ -174,3 +174,65 @@ def test_synapse_modoverride(create_tmp_simulation_config_file):
         for nc_id, nc in enumerate(nclist):
             utils.check_netcon(src_gid, nc_id, nc, edges, selection, **kwargs)
             utils.check_synapse(nc.syn(), edges, selection, **kwargs)
+
+
+@pytest.mark.parametrize("create_tmp_simulation_config_file", [
+    {
+        "simconfig_fixture": "ringtest_baseconfig",
+        "extra_config": {
+            "target_simulator": "NEURON",
+            "node_set": "Mosaic",
+            "connection_overrides": [
+                {
+                    "name": "A2B",
+                    "source": "RingA",
+                    "target": "RingB",
+                    "spont_minis": 200
+                }
+            ]
+        }
+    },
+], indirect=True)
+def test_spont_minis_simple(create_tmp_simulation_config_file):
+    """Test that spont_mini fires with roughly """
+    from neurodamus import Neurodamus
+    from neurodamus.core import NeurodamusCore as Ndc
+    from neurodamus.connection import NetConType, SpontMinis
+
+    src_pop, src_gid, tgt_pop, tgt_gid = "RingA", 1, "RingB", 1001
+
+    nd = Neurodamus(create_tmp_simulation_config_file)
+    edges_a_b = nd.circuits.get_edge_manager(src_pop, tgt_pop)
+    # nd.connections not netcon
+    # this should retrieve still only 1 connection. The
+    # spont_minis attaches an orphan netcon directly to the
+    # synapse. Here we check that the old netcon is there
+    # and the spont_minis is placed
+    connections = list(edges_a_b.get_connections(tgt_gid))
+    assert len(connections) == 1
+    conn = next(iter(connections))
+    # old netcon is still up and active right?
+    assert conn.sgid == src_gid
+    assert conn.tgid == tgt_gid
+    # check that there is also a spont_minis attached
+    # to this synapse
+    assert conn.minis_spont_rate == 200
+    assert isinstance(conn._spont_minis, SpontMinis)
+    assert len(list(conn._spont_minis.netcons)) == 1
+    nc = next(iter(conn._spont_minis.netcons))
```

Comment:
How about checking netcon in the neuron cvode context? In that way, we are sure the netcons will be sent to neuron.

Reply:
the spont mini creates a new netcon that does not have a source. How would you get the netcon from cvode context? I can try, maybe I did not find the correct neuron context yet

Reply:
yes, in this case `netcon.srcgid()` shall be negative, @juanjosegarcan  found it out for ReplayStim netcons. We can do similar as 
https://github.com/openbraininstitute/neurodamus/blob/ae27d40ac931e49cc122353f08d2111a7bb9c772/tests/unit/test_replay_manager.py#L175
check `netcon.srcgid()` is negative and `netcon.weight[4] == int(NetConType.NC_SPONTMINI)`, as well as the delay and `netcon.weight[0]`